lib/task/db.rake

```diff
@@ -9,12 +12,13 @@ task :create_test_db do
   conn = PG.connect(
     host: uri.hostname,
     port: uri.port,
-    dbname: 'postgres',
+    dbname: uri.scheme,
```

Comment:
What's the relation between the URL scheme and database name?

Reply:
Looks like [Rails uses it](https://github.com/rails/rails/blob/feb0fd0ca91052bbdce8ed67ffa937ecfcf249ec/activerecord/lib/active_record/connection_adapters/connection_specification.rb#L41). Funny thing is that 'posgres' is the only exceptional case.

Anyway, I changed to use Rails APIs to connect, drop and create the database. :tada: